testing/trino-testing/src/main/java/io/trino/testing/BaseOrcWithBloomFiltersTest.java

```diff
@@ -61,23 +50,14 @@ public void testOrcBloomFilterIsWrittenDuringInsert()
                 format(
                         "CREATE TABLE %s (totalprice DOUBLE, orderstatus VARCHAR) WITH (%s)",
                         tableName,
-                        getTableProperties("totalprice", "orderstatus", "totalprice")));
```

Comment:
I believe the sort by totalprice exists so that the condition `totalprice 51890 is chosen to lie between min/max values of row group` will be satisfied. Without explicit ordering this is not guaranteed.
fyi @Praveen2112

Reply:
I've tested it. It can verify bloom filter without sorting.

Reply:
Because only one Orc file is generated, and the number of records is only 15000 lines. Sorting or not has no effect.

Reply:
> I guess it is better to have `totalprice` sorted instead of removing.

Done. I added the order by option when CTAS and Insert INTO SELECT .

Reply:
i am reopening this conversation, because the alternative approach isn't OK. Please keep `sorted_by=totalprice`  for Hive.

We can think what are our options for Iceberg separately.

(**IF** sorting for Hive isn't needed, we can remove it, but that would be a separate prep commit. I am **not** saying it is not needed.)

Reply:
> i am reopening this conversation, because the alternative approach isn't OK. Please keep `sorted_by=totalprice` for Hive.
> 
> We can think what are our options for Iceberg separately.
> 
> (**IF** sorting for Hive isn't needed, we can remove it, but that would be a separate prep commit. I am **not** saying it is not needed.)

Done.Back to the original